open-craft/opencraft
# [BB-3786] Port Logo Page to the overlay menu

We're fulfilling the following user story:

>"As an OpenCraft Developer, I want to port the logo update page to the new console"

**References**: [BB-3786](https://tasks.opencraft.com/browse/BB-3786), [Gitlab issue](https://gitlab.com/opencraft/dev/opencraft/-/issues/714)

**Screenshots**: 
![image](https://user-images.githubusercontent.com/7670449/114147287-2f8d3880-9936-11eb-9719-b2948537b309.png)


**Merge deadline**: "None" 

**Testing instructions**:

1. Pull this branch to your local OCIM environment.
2. Go to http://localhost:3000/newconsole/theming/logos and check that it shows the same behavior as http://localhost:3000/console/theming/logos

**Author notes and concerns**:

1. After development is done, we should have a large stylings review to address responsive design.
2. At some point, there should be a correctness test (these tend to depend on the backend, try uploading a large logo and a large favicon)

**Reviewers**
